client/my-sites/customer-home/main.jsx

```diff
@@ -509,6 +510,31 @@ class Home extends Component {
 						</Card>
 					) }
 					{ ! siteIsUnlaunched && <StatsCard /> }
+					<Card className="customer-home__free-photo-library-card">
+						<video
```

Comment:
Since the video is showing a desktop interface, I'd suggest to hide it on mobile viewports. That way we can also save some height on small screens.

@Automattic/manage should we be concerned about non-English users? Video is quite simple and I think most of the users should be able to follow it despite showing an English interface.

Reply:
That seems reasonable! What's the preferable approach to hiding things on mobile with Customer Home? Usually it's CSS, but I find many cases of `isMobile()` in this file.

Reply:
For things that need to be hidden is better to use `isMobile` since that will prevent unnecessary elements from being rendered in the DOM. CSS media queries are great for tweaking styles, but if something is not going to be visible, let's not render them and make the DOM lighter.

Reply:
Okay! Added a commit to hide the video on mobile. :)